app/connectors/ContactFrontendConnector.scala

```diff
@@ -40,7 +40,7 @@ class ContactFrontendConnector @Inject()(
     client.GET[String](url) recover {
       case ex: BadGatewayException =>
         Logger.error(
-          s"[ContactFrontendConnector][getHelpPartial] an exception occurred ${ex.message} when calling $url",
+          s"[ContactFrontendConnector][getHelpPartial] A bad gateway exception occurred when calling $url - ${ex.message}",
```

Comment:
Maybe doesn't apply here since it gets named in the error message, but I think there's value in logging the whole exception rather than just the message. Sometimes knowing the exception type helps investigation.

Reply:
My bad - could swear I added it there. Changing now

Reply:
It's already logging the whole exception - is line 44 what you mean?

Reply:
oh I missed that woopsie!